packages/spec/integration/esbuild/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "fixture-esbuild",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "12 || 14 || 15"
+  },
+  "hops": {
+    "gracePeriod": 0
+  },
+  "jest": {
+    "displayName": "integration",
+    "testEnvironment": "jest-environment-hops",
+    "setupFilesAfterEnv": [
+      "../../jest.setup.js"
+    ]
+  },
+  "scripts": {
+    "start": "hops start",
+    "build": "hops build"
```

```suggestion
    "start": "hops start --experimental-esbuild",
    "build": "hops build --experimental-esbuild"
```
It's a small thing though and not really required.